### PR TITLE
[Buttons] overwrite intrinsicContentSize method.

### PR DESCRIPTION
**Pull request by fluxminds**
The `instrinsicContentSize` of `MDCButton` was not aware of `minimumSize` property, which is wrong.

Open to use `super.instrinsicContentSize` if reviewer(s) prefer.

### CODE REVIEW

**Review by romoore — CHANGES_REQUESTED** [review]
Please add tests for the new behavior.

**Comment by fluxminds** [comment]
Added tests and removed WIP from title.

**Review by randallli — APPROVED** [review]

File: components/Buttons/tests/unit/MDCButtonTests.m [hunk 1]
```diff
@@ -1276,6 +1276,43 @@ - (void)testSizeThatFitsWithMinimumAndMaximum {
                 NSStringFromCGRect(expectedFrame), NSStringFromCGRect(self.button.frame));
 }
 
+- (void)testIntrinsicContentSizeWithMinimum {
```
Comment (romoore) [inline components/Buttons/tests/unit/MDCButtonTests.m]: Tip: Please describe the "outcome" in the test name.  Here and in the others.

**Review by romoore — APPROVED** [review]

File: components/Buttons/tests/unit/MDCButtonTests.m (hunk 1 above)
Comment (fluxminds) [reply components/Buttons/tests/unit/MDCButtonTests.m]: Addressed in df99798

**Pull request merged**